ts/Core/Time.ts

```diff
@@ -409,19 +408,30 @@ class Time {
         }
 
         if (options.timezone) {
-            if (!moment) {
-                // getTimezoneOffset-function stays undefined because it depends
-                // on Moment.js
-                error(25);
-
-            } else {
-                return function (timestamp: (number|Date)): number {
-                    return -moment.tz(
-                        timestamp,
-                        options.timezone
-                    ).utcOffset() * 60000;
-                };
-            }
+            return (timestamp: number | Date): number => {
+
+                try {
+                    // eslint-disable-next-line new-cap
+                    const hourOffset = Intl.DateTimeFormat('en', {
+                        timeZone: options.timezone,
+                        timeZoneName: 'shortOffset'
+                    })
+                        .format(timestamp)
+                        .split('GMT')[1]
+                        .replace(':30', '.5');
+
+                    // Make sure only valid numbers are returned, otherwise fall
+                    // back to 0. Safety vent for avoiding full crash in future
+                    // cases that we don't yet know about.
```

Comment:
When checking the [list](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), it seems like there might be an issue with [Nepal](https://en.wikipedia.org/wiki/UTC%2B05:45) (`Asia/Katmandu`) and possibly with some historical stuff (e.g. [this](https://en.wikipedia.org/wiki/UTC%2B01:24) or [that](https://en.wikipedia.org/wiki/UTC%2B04:51)).

Reply:
Thanks for taking the time to test thoroughly! I have a fix for Nepal. The two last ones don't seem to be known by browsers.